README.md

```diff
@@ -3,10 +3,10 @@
 ## Table of Contents
 
 1. [Introduction](#introduction)
-1. [Terms and Definitions](#terms-and-definitions)
-1. [SecAPI Overview](#secapi-overview)
-1. [Sample Use Cases](#sample-use-cases)
-1. [Robustness Rules](#robustness-rules)
+2. [Terms and Definitions](#terms-and-definitions)
```

Comment:
Is there a markdown style format spec which is driving these changes? If so, let's add a Reference section to this document and add a link to the style spec.

Reply:
We should call out the Doxygen style expectation/requirement in this document. Otherwise, some future person editing the README will have to learn the hard way.

Reply:
Done